server/data/quotes.json

```diff
@@ -48,7 +48,7 @@
     "sentence": "One must not be shy where language is concerned."
   },
   {
-    "author": "Anne Sexton",
+    "author": "Anne **ton",
```

Comment:
No need to do this. Let the poet's name be there as it is.

Reply:
Sorry for this, it might have occured automatically by my system word restrictor. I didnt mean to change that.